shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -121,26 +133,75 @@ def dicom_to_nifti(path_dicom, path_nifti, subject_id='sub-01', fname_config_dcm
             with open(fname_json_b1) as json_file:
                 json_data = json.load(json_file)
                 # Check what B1+ mapping sequence has been used to proceed accordingly
-                # If Siemens' TurboFLASH B1 mapping (dcm2niix cannot separate phase and magnitude for this sequence)
                 if ('SequenceName' in json_data) and 'tfl2d1_16' in json_data['SequenceName']:
-                    fname_nii_b1 = fname_json_b1.split(".json")[0] + ".nii.gz"
-                    nii_b1 = nib.load(fname_nii_b1)
-                    nii_b1_new = fix_tfl_b1(nii_b1, json_data)
-
-                    # Save uncombined B1+ maps in a NIfTI file that can now be visualized in FSLeyes
-                    json_data["ImageComments"] = 'Complex uncombined B1+ map (nT/V)'
-                    fname_nii_b1_new = fname_nii_b1.split('.nii')[0] + '_uncombined.nii' + fname_nii_b1.split('.nii')[1]
-                    nib.save(nii_b1_new, fname_nii_b1_new)
-
-                    # Save the associated JSON file
-                    fname_json_b1_new = open(os.path.join(fname_nii_b1_new.split('.nii')[0] + '.json'), mode='w')
-                    json.dump(json_data, fname_json_b1_new)
-
-                    # Remove the old buggy NIfTI and associated JSON files
-                    os.remove(fname_nii_b1)
-                    os.remove(fname_json_b1)
-                # TODO: Add handling of other B1+ mapping sequences
 
+                    # If NeuroSpin B1 mapping (interferometric sequence)
+                    if ('SeriesDescription' in json_data) and \
+                       ('ns_tfl_rfmap_interferometric' in json_data['SeriesDescription']):
```

Comment:
```suggestion
                       ('ns_tfl_rfmap_interferometric' in json_data['ProtocolName']):
```

According to:
https://github.com/shimming-toolbox/shimming-toolbox/blob/ce256489659ab01a61581e0762d1777c8f34038b/config/dcm2bids.json#L281